sdk/lib/crt/math/arm/__fto64_worker.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of __dtoi64
+ * COPYRIGHT:   Copyright 2021 Roman Masanin <36927roma@gmail.com>
+ */
+
+#ifdef _USE_64_BITS_
+typedef unsigned long long FINT_TYPE;
+#define FRACTION_LEN 52
+#define EXPONENT_LEN 11  
+#else
+typedef unsigned int FINT_TYPE;
+#define FRACTION_LEN 23
+#define EXPONENT_LEN 8   
+#endif
+
+typedef
+#ifndef _USE_SIGNED_
+unsigned
+#endif 
+long long FTO64_RESULT;
+
+#define INTNAN ((FINT_TYPE)1) << (FRACTION_LEN + EXPONENT_LEN)
+
+#define SIGN_MASK ((FINT_TYPE)1) << (FRACTION_LEN + EXPONENT_LEN)
+
+#define FRACTION_ONE ((FINT_TYPE)1) << FRACTION_LEN
+#define FRACTION_MASK (FRACTION_ONE) - 1
+
+#define EXPONENT_MASK (1 << EXPONENT_LEN) - 1
+#define EXPONENT_ZERO (1 << (EXPONENT_LEN - 1)) - 1
+#define EXPONENT_MAX (EXPONENT_LEN - 1) + FRACTION_LEN
+
+#define EXPONENT_INFINITY EXPONENT_MASK
+
+#define NEGATE(x) x = ~x + 1
+
+FTO64_RESULT
+__fto64_worker(FINT_TYPE value)
+{
+    int exponent;
+    FTO64_RESULT fraction;
+    
+    exponent = (int)(value >> FRACTION_LEN);
+    exponent &= EXPONENT_MASK;
+    
+    /* infinity and other NaNs */
+    if (exponent == EXPONENT_INFINITY)
+        return INTNAN;
+            
+    /* subnormals and signed zeros */
+    if (exponent == 0)
+        return 0;
+
+    exponent -= EXPONENT_ZERO;
+
+    /* number is less then one */
+    if (exponent < 0)
+        return 0;
+
+    /* number is too big */
+    if (exponent > EXPONENT_MAX)
+        return INTNAN;
+    
+    fraction = value & FRACTION_MASK;
+    fraction |= FRACTION_ONE;
+    
+    exponent -= FRACTION_LEN;
+    if ( exponent > 0)
```

Comment:
one excessive space :smile: 
```suggestion
    if (exponent > 0)
```